### ISSUE TITLE
adopt starscream 3.0.3

### ISSUE THREAD

**Issue by piemonte** [florianmari/LiveGQL#7]
great library! 🙌 would be nice to support the latest starscream.

**Comment by piemonte** [comment]
```
/Pods/Starscream/Source/WebSocket.swift:122:28: warning: when calling this function in Swift 4 or later, you must pass a '()' tuple; did you mean for the input type to be '()'?
    public var onConnect: ((Void) -> Void)?
                           ^~~~~~
                           ()
/Pods/Starscream/Source/WebSocket.swift:553:34: error: missing argument for parameter #1 in call
                    s.onConnect?()
                                 ^
                                 <#Void#>
```

**Comment by florianmari** [comment]
Hey thanks for your issue, I'm working on a new version!

**Comment by florianmari** [comment]
Hello, a new version is now available! 2.0.0

**Comment by piemonte** [comment]
@florianmari awesome, thanks much!
